app/views/staff/tracks/update.js.erb

```diff
@@ -1,4 +1,10 @@
-$('#config-modal').modal('hide');
+<% if @track.save %>
```

Comment:
I have that in all the modals (@room.save, @session_format.save) - my intention was just to check if it *had* saved - if @track.save returns a boolean..